core/broadleaf-framework-web/pom.xml

```diff
@@ -98,5 +98,9 @@
             <groupId>org.easymock</groupId>
             <artifactId>easymockclassextension</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I see you've added a new dependency, but what about deleting old spring social ? Is it still there ?